assets/js/components/ExecutionResults/checksUtils.js

```diff
@@ -102,6 +102,14 @@ export const getCheckHealthByAgent = (checkResults, checkID, agentID) => {
   };
 };
 
+export const isPremium = (catalog, checkID) => {
```

Comment:
Could we add a test on this? I know it is tested indirectly in other tests, but always good to have unit testing

Reply:
Yes, sure!